suite-common/wallet-core/src/token-definitions/tokenDefinitionsMiddleware.ts

```diff
@@ -1,87 +1,33 @@
-import { isAnyOf } from '@reduxjs/toolkit';
```

Comment:
Please move all these redux files to tokens-definitions package, it's not something that should be in core and it should be together with other token definitions things.

Reply:
Why, it is part of `wallet` state

Reply:
I don't think it should be part of wallet state, it's config same as message system. We should keep wallet-core thin as possible or it will lead to huge monolithic package again.

Reply:
True, I agree, let's leave it for follow up please

Reply:
nope it is not saved to storage

Reply:
https://github.com/trezor/trezor-suite/issues/11506